launch_test.go

```diff
@@ -228,3 +230,51 @@ func TestClone(t *testing.T) {
 	}
 	launch(screwdriver.API(api), testBuildID)
 }
+
+func TestCheckIfPR(t *testing.T) {
+	testJobName := "PR-1"
+	wantPrNumber := "1"
+	matchString = func(pattern string, jobName string) (bool, error) {
+		if jobName != testJobName {
+			t.Errorf("jobName == %q, want %q", jobName, testJobName)
+		}
+		return true, nil
+	}
+
+	prNumber, err := checkIfPR(testJobName)
+	if err != nil {
+		t.Errorf("Unexpected error checking pull request")
+	}
+
+	if prNumber != wantPrNumber {
+		t.Errorf("prNumber == %q, want %q", prNumber, wantPrNumber)
+	}
+}
+
+func TestFetchAndMerge(t *testing.T) {
+	testBuildID := "BUILDID"
```

Comment:
Split tests? Feels redundant to do so and doesn't generate further code coverage.